### PR TITLE
🤗 Remove EP dependencies from OptimumMerge

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes

Remove EP config from OptimumMerge and leverage the accelerators info from accelerator_spec.

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Review by devang-ml — DISMISSED** [review]

File: olive/passes/olive_pass.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def __init__(
 
         config_class, default_config = self.get_config_class(accelerator_spec, disable_search)
 
-        self._accelerator_spec = accelerator_spec
+        self.accelerator_spec = accelerator_spec
```
Comment (xiaoyu-work) [inline olive/passes/olive_pass.py:77]: Any reason for this change?

**Comment by xiaoyu-work** [comment]
Can you explain some backgrounds about this change?

**Review by xiaoyu-work — APPROVED** [review]

**Comment by trajepl** [comment]
> Can you explain some backgrounds about this change?

https://github.com/microsoft/Olive/pull/434 

Basically it is used to remove the EP configs from optimum merging pass. Because we already have the accelerator_spec into pass which contains the EP information already(`self.accelerator_spec`).

**Pull request merged**